src/login.py

```diff
@@ -1,7 +1,18 @@
+import re
+
 class Login:
     def validate(self, form):
+        if 'name' not in form or 'mobile' not in form or 'email' not in form:
+            return False
+
         email = form['email']
-        if not isinstance(email, str):
+        name = form['name']
+        mobile = form['mobile']
+
+        if not isinstance(email, str) or not isinstance(name, str) or not name or not email:
             return False
-        else:
-            return True
+
+        if len(email) > 7:
```

Comment:
Might be a good idea to move this to a separate function `is_valid_email`. Also let's create a constant EMAIL_LENGTH for this magic number 7?